src/server.cpp

```diff
@@ -440,6 +440,7 @@ void Server::init()
 
 	m_script->loadMod(getBuiltinLuaPath() + DIR_DELIM "init.lua", BUILTIN_MOD_NAME);
 
+	m_gamespec.checkAndLog();
```

Comment:
this feels a bit randomly placed, where is m_gamespec read/initialized?

Reply:
It's here because the next line loads the mods and calls `m_modspec.checkAndLog()`